examples/basic/basic-connect.js

```diff
@@ -11,5 +15,5 @@ client.connect()
   })
   .catch(function (err) {
     console.error('There was an error when connecting', err);
-    return client.shutdown();
+    return client.shutdown().then(() => process.exit(1));
```

Comment:
I noticed that runner.js was marking examples as passed even though they were actually failing.  Registering a non zero exit code in these cases did the trick.